testsuite/forge-cli/src/main.rs

```diff
@@ -525,32 +529,27 @@ fn single_test_suite(
     duration: Duration,
     test_cmd: &TestCommand,
 ) -> Result<ForgeConfig> {
+    // Check the test name against grouped test suites
+    if let Some(forge_config) = get_land_blocking_test(test_name, duration, test_cmd) {
```

Comment:
this to me is less clear than the previous way that matched names to setup functions. If we want to have some kind of grouping that was already being done with comments interspersed in the list.

Reply:
I think this is debatable. 😄 The issue I see is that we're going to keep adding to this list (in fact, I have several sets of network tests to add, as well as PFN tests, etc.). So, unless folks are strict in keeping the list tidy, and ordered (e.g., alphabetically or by functionality?), it is going to get very unwieldy quickly. In fact, I'd argue it already is.

If we see a better approach, I'm happy to follow it, but given that this code is "unowned", it tends to messy very fast. At least with the grouping, it's easier to split things up.

Reply:
FWIW, I've added a comment to the function. Hopefully someone can propose/implement a cleaner solution 😄